src/components/screen-header/ScreenHeader.tsx

```diff
@@ -180,13 +173,16 @@ const useHeaderStyle = StyleSheet.createThemeHook((theme) => ({
     paddingBottom: theme.spacings.medium,
     borderTopLeftRadius: theme.border.radius.circle,
     borderTopRightRadius: theme.border.radius.circle,
+    marginBottom: theme.spacings.medium,
+    alignItems: 'center',
   },
-  headerTitle: {alignItems: 'center'},
+  headerTitle: {alignItems: 'center', width: '66%'},
```

Comment:
How is 66% decided?

Reply:
I did some testing and figured that 2/3 of the screen made sense for the title. Might be a better value to use here, but it has to be a percentage so that zooming works nicely. Any suggestions for a better width or how to find one?

Reply:
Nope, just curious! As long as it works with 200% zoom on devices of various widths it should be alright.